news/2022-12-01-osumonthly-5.md

```diff
@@ -0,0 +1,184 @@
+
+### SillyFanGirl's BR eSports nomination
+
+![https://twitter.com/premioesportsbr/status/1593590309631295488?s=20&t=1thfKY6-BpTCFDsbtltDdw](https://pbs.twimg.com/profile_banners/983759809156669440/1663599648/1500x500)
+
+It is already a very hard feat to be recognized as one of the best, most versatile players 4K osu!mania has ever seen. [SillyFanGirl](https://osu.ppy.sh/users/2288363) has not only been able to achieve this, but is also an active [content creator](https://www.youtube.com/@SillyFangirlosu), an active tournament player, and overall an incredible community member to interact with!
+
+But as if their astounding performance on [this year's osu!mania World Cup](/wiki/Tournaments/MWC/2022_4K) was not enough, for the second consecutive year they've been nominated in Brazil's eSports Award in the "Best Athlete in Other Modalities" category!
+
+If you want to directly support them or get yourself involved in the Brazil's eSports Awards show, [we encourage you to do so by participating on their website](https://peb.gg).
+
+### Notable osu! plays of the month
+
+*Section written by [0x84f](https://osu.ppy.sh/users/7944724)*
+
+Month after month [mrekk](https://osu.ppy.sh/users/7562902) keeps proving why he is the best osu! player in the world, and if you *still* needed some justification as to why people consider him as one of the greatest players of all time, check out his HDDT play on [Save Me](https://osu.ppy.sh/beatmapsets/594170#osu/1256809). This 11 minute long marathon by [Drummer](https://osu.ppy.sh/users/4102249) has been considered by some as the "endgame" map to FC with HDDT, unfortunately mrekk choked at the end with 4 misses and a couple of slider breaks but nonetheless, an amazing play to witness.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/SSJOtgLmOIo" frameborder="0" allowfullscreen></iframe>
+
+Another notable play amongst a sea of mrekk's scores that push the game beyond it's limits would be [worst hr player](https://osu.ppy.sh/users/14106450/)'s 1x sliderbreak run on [Katagiri](https://osu.ppy.sh/beatmaps/artists/176)'s remix of [true DJ MAG top ranker's song Zenpen](https://osu.ppy.sh/beatmapsets/1357624#osu/2809623), which you can [check out here](https://www.youtube.com/watch?v=qKOefiZZT8U).
+
+### Notable osu!taiko plays of the month
+
+*Section written by [Blushing](https://osu.ppy.sh/users/5927823)*
+
+It's one thing to get the number one spot on the leaderboard, but it's another to completely be the only FC with a mod at all, lest it be hidden of all mods. Though, [Peaceful](https://osu.ppy.sh/users/165027) took up the challenge and managed to absolutely *defile* and *violate* [The Violation](https://osu.ppy.sh/beatmapsets/1452092#taiko/2985808).
```

Comment:
"defile" and "violate" were used together because of the difficulty name "Defilement" + song name "The Violation", other changes are OK but let me know if you are OK with them two staying or if you want only "violate" to stay.

Reply:
ok yeah i didn't notice the Defilement diff name, but in this sentence you'd have know that to understand that "defile" would be relevant so i'd personally still exclude it.

if you worded it like "absolutely *defile* and *violate* the Defilement diff of [The Violation](https://osu.ppy.sh/beatmapsets/1452092#taiko/2985808)" i'd be more ok with it, but i don't think it sounds as neat

stylistic stuff

Reply:
@Blooshing can you review Wala's comments and let me know how you'd like to proceed

Reply:
Let's word it like wala, I wanna keep the play on words in the sentence if possible.

Reply:
I will apply Wala's original different diff change since saying the "Defilement diff of The Violation" implies there's more than one diff